src/live_restore/live_restore_fs.c

```diff
@@ -908,6 +1028,7 @@ __live_restore_fh_find_holes_in_dest_file(
         WT_ERR(__live_restore_remove_extlist_hole(
           lr_fh, session, data_offset, (size_t)(data_end_offset - data_offset)));
     }
+    __wt_writeunlock(session, &lr_fh->ext_lock);
```

Comment:
Should this move to under the err: label?

Reply:
Yes: c1db045ed